1주차/1316.py

```diff
@@ -0,0 +1,20 @@
+N = int(input()) #단어 수
+count = 0 #그룹단어 수
+
+
+for _ in range(N):
+    word = input() #N개만큼 단어 입력
+    previous_letter = [] #전에 나온 알파벳을 저장하는 곳
+    group_word = "1"
```

Comment:
```suggestion
    is_group_word = True
```
그룹 단어인지 여부를 체크하는 변수는 문자열 "1", "0"보단 이런식으로  불리언값으로 체크하는게 더 이해하기 좋을 것 같아요